test/integration/mongodb/mongodb.test.js

```diff
@@ -0,0 +1,492 @@
+const {test, describe} = require("../integrationTestUtils/setup");
+const fs = require("fs");
+const waitOn = require('wait-on')
+require("jest-json");
+
+const {waitForSpansInFile} = require("../integrationTestUtils/waiters");
+const {callContainer} = require("../../helpers/helpers");
+const {spawn} = require("child_process");
+const kill = require("tree-kill");
+
+const SPANS_DIR = `${__dirname}/spans`;
+const EXEC_SERVER_FOLDER = "test/integration/mongodb/app";
+const TEST_TIMEOUT = 300000;
+const WAIT_ON_TIMEOUT = 80000;
+const APP_PORT = 8080;
+const INTEGRATION_NAME = `mongodb`;
+
+function getSpanByName(spans, spanName) {
+    return spans.find((span) => span.name === spanName);
+}
+
+function isAllSpansInFile(filePath) {
```

Comment:
The function name should be something like `getAllSpansInFile` or the return type should be `Boolean`